docs/CODE_REVIEW.md

```diff
@@ -87,17 +87,16 @@ We wouldn’t want anyone being mean to us because of an oversight, mistake, or
 
 We use this list when performing a code review to ensure that all tasks have been completed.
 
-- [ ] begin by deleting any existing copy of the Git repository, running `docker system prune` to remove any cached artifacts from any prior code review
-- [ ] clone a fresh copy of the Git repository (`git clone https://github.com/ustaxcourt/ef-cms.git`), fetch the pull request and name it for the sprint (e.g., `git fetch origin pull/204/head:sprint-43`), and then switch to that branch (e.g. `git checkout sprint-43`)
+- [ ] Fetch the pull request for the sprint (e.g., `git fetch origin pull/204/head:sprint-43`), and then switch to that branch (e.g. `git checkout sprint-43`)
```

Comment:
I actually do run `docker system prune`, mostly because if I don't do that, I might have a lot of storage space taken up by older, unused image builds.

Reply:
Added a discretionary step in a2ed1d4: 

> If desired, run `docker system prune` to remove any unused docker images from previous code review.